### PR TITLE
Removal of FusedOp inheritance leftovers

### PR DESCRIPTION
**Pull request by mitruska**
### Details:
 - Removal of FusedOp inheritance from the remaining ops
   (v0::Gelu, v0::MVN, v0::HardSigmoid, v0::LSTMSequence)

- Enable Gelu type_prop tests (missed in CMakeLists.txt) 

### Tickets:
 - 62747, related to 30457

### CODE REVIEW

**Review by ggalieroc — APPROVED** [review]

**Comment by ilyachur** [comment]
Can we remove `FusedOp`?

**Comment by mitruska** [comment]
> Can we remove `FusedOp`? Or we still have some fused operations.

Yes, it seems finally we can. There are no more operations with `FusedOp` as base class.

File: ngraph/core/src/op/gelu.cpp [hunk 1]
```diff
@@ -63,7 +35,7 @@ shared_ptr<Node> op::v0::Gelu::clone_with_new_inputs(const OutputVector& new_arg
     return make_shared<op::v0::Gelu>(new_args.at(0));
 }
 
-void op::v0::Gelu::pre_validate_and_infer_types() {
+void op::v0::Gelu::validate_and_infer_types() {
```
Comment (bszmelcz) [inline ngraph/core/src/op/gelu.cpp:38]: shouldn't be NGRAPH_OP_SCOPE added here?

Comment (mitruska) [reply ngraph/core/src/op/gelu.cpp:38]: Ok, Done (for HardSigmoid also)

**Review by ilyachur — APPROVED** [review]

**Review by bszmelcz — APPROVED** [review]

**Pull request merged**